cluster/gce/gci/configure-helper.sh

```diff
@@ -2423,7 +2440,7 @@ EOF
   fi
   if [[ "${ENABLE_NODE_TERMINATION_HANDLER:-}" == "true" ]]; then
       setup-addon-manifests "addons" "node-termination-handler"
-      setup-node-termination-handler-manifest
+      setup-node-termination-handler-manifest "$@"
```

Comment:
Where is $@ being populated from? Can we make this a little more clear? start-kube-addons does not seem to be taking an arguments.

Reply:
Yes, it is unclear if it is ok to call setup-node-termination-handler-manifest without any arguments. I guess

`local -r nth_manifest="/etc/kubernetes/$1/$2/daemonset.yaml"`

would still work without $1 and $2, so no harm done if that is what was intended.
I think to keep the current behavior, this line should be

`setup-node-termination-handler-manifest '' ''`

And then an eventual refactorization is to be left as as separate task.